lib/mcb/commands/providers/edit.rb

```diff
@@ -1,11 +1,36 @@
 summary "Edit information about provider"
-param :code, transform: ->(code) { code.upcase }
+option nil, :'accrediting-provider',
+       "Update the accrediting provider attribute to accredited_body.",
+       default: false
+option nil, :'not-accrediting-provider',
+       "Update the accrediting provider attribute to not_an_accredited_body.",
+       default: false
+
+
 
 run do |opts, args, _cmd|
   MCB.init_rails(opts)
+  recruitment_cycle = MCB.get_recruitment_cycle(opts)
+  providers = recruitment_cycle.providers
+
+  if opts[:"accrediting-provider"].present?
```

Comment:
Looks good but [noob question] is there any particular reason why it's not part of the editor wizard?

Reply:
> [noob question] is there any particular reason why updating accrediting providers is not part of the editor wizard?

@TobyRet I think we had/have a ticket for doing that, but I haven't been able to find it.